### DISCUSSION TITLE
Dew Heater

### DISCUSSION THREAD

**Discussion by Dan-Judd** [aaronwmorris/indi-allsky#1742] [Q&A] [answered]
I have my dew heater put together and working independent of Indi-Allsky. My DHT22 is working reporting Air Temp and RH. My goal is to have Indi-Allsky turn on the dew heater any time day or night when the RH gets above 60%. Unfortunately, covering the DHT22 with a damp paper towel does not turn on  GPIO pin 17 when the RH exceeds 60%. While experimenting with different settings I did find that my GPIO pin goes high if I turn on Enable Dew Heater Thresholds. Here is how I have the Dew Heater setting configured:
![image](https://github.com/user-attachments/assets/3bc0e949-b0a7-49c3-8705-51254396cfad)

One other question. I noticed in another discussion someone had the dew heater status showing on the Overlay. What is the variable name used for this?

Once again, thanks for your help.

**Comment by aaronwmorris** [comment]
The dew heater status will be `{dew_heater_status:s}`.

Based on your settings, I do not think the dew heater would ever turn off because your default level is 100.  The note is beside the low threshold difference:  `If the difference is above the LOW threshold, the default level is used`.

Are you sure you have the dew heater hooked up to the correct pin?

**Comment by Dan-Judd** [comment]
Pin is correct. With Default level at 100 and Enable Dew Heater Thresholds ON, GPIO pin 17 is ON. With Default Level set to 0 or Thresholds OFF, GPIO pin 17 is OFF. So in a round about way I can turn the dew heater on and off at this point.

I was under the impression that Default Level and all the thresholds and Low Mid and High Settings as well as the Low Mid and High Thresholds were only used in PWM mode, but that is obviously not the case. How do these variables affect Dew Heater Standard mode?

**Reply by aaronwmorris** [reply, marked as answer]
With the Standard Dew Heater mode, any non-zero value means the dew heater would be On.

**Comment by Dan-Judd** [comment]
It took a bit to wrap my head around it, but I got it working. Thanks for your help.
